ui/src/app/pages/profile/demographic-survey.tsx

```diff
@@ -160,7 +160,7 @@ export class DemographicSurvey extends React.Component<Props, State> {
 
     return <div style={{marginTop: '1rem', paddingLeft: '1rem', width: '32rem'}}>
       <TextColumn>
-        <div style={{fontSize: 28, fontWeight: 400, marginBottom: '.8rem'}}>Optional Demographics Survey</div>
+        <div style={{fontSize: 28, fontWeight: 400, marginBottom: '.8rem'}}>Demographics Survey</div>
         {this.props.showStepCount &&
           <div style={{fontSize: 16, marginBottom: '.5rem'}}>
             Please complete Step '3 of 3'
```

Comment:
While you're making text updates, do you mind dropping the quotes here? Note: the previous pages do not include these single quotes